### ISSUE TITLE
MIT license 

### ISSUE THREAD

**Issue by aspina7** [R4EPI/sitrep#146]
> without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, _**and/or sell copies**_ of the Software

Would be nice to have a license that doesnt allow people to just swipe the whole repo and sell it as is. 
Something where people are allowed to use, copy and share alike but not for commercial purposed as a product in itself. (i.e. can be used for commercial purposes but must be packaged within a seperate project or training....)

**Comment by zkamvar** [comment]
IMO, the *concept* of this is more threatening than the reality. Most licenses allow some form of commercial venture, and MIT is the most permissive. I think the one you might be seeking is GPLv3: https://choosealicense.com/licenses/

**Comment by aspina7** [comment]
note may want to add multiple copyright holders - in this case zhian and annick? https://github.com/R4EPI/sitrep/issues/44#issuecomment-459341997

**Comment by dirkschumacher** [comment]
No, do not limit commercial use. It just makes stuff more complicated. Is Government commercial? Is a contractor working for MSF commerical? I think GPL or MIT are the 2 viable options in the R space at the moment.
